kafka/source/pkg/reconciler/kafkasource.go

```diff
@@ -206,6 +219,188 @@ func checkResourcesStatus(src *v1alpha1.KafkaSource) error {
 	return nil
 }
 
+func (r *Reconciler) scaleKafkaSource(ctx context.Context, ra *v1.Deployment, src *v1alpha1.KafkaSource) {
+	r.readScalingAnnotations(ctx, src)
+	// no scaling annotatins so no scaling work needed
+	if r.minReplicaCount == nil && r.maxReplicaCount == nil {
+		return
+	}
+	_, error := r.deployKedaScaledObject(ctx, ra, src)
+	if error != nil {
+		// additional logging?
```

Comment:
Yes please, lets log on either success or failure from `deployKedaScaledObject` (perhaps different log levels so that success is less noisy)

Reply:
fixed to propagate to reconcile